src/MongoDB.Driver.Core/ReadPreferenceHedge.cs

```diff
@@ -0,0 +1,142 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using MongoDB.Bson;
+
+namespace MongoDB.Driver
+{
+    /// <summary>
+    /// Represents the read preference hedge.
+    /// </summary>
+    public abstract class ReadPreferenceHedge : IEquatable<ReadPreferenceHedge>
+    {
+        #region static
+        // private static fields
+        private static readonly CustomReadPreferenceHedge __enabled = new CustomReadPreferenceHedge(isEnabled: true);
+        private static readonly ReadPreferenceHedge __serverDefault = new ServerDefaultReadPreferenceHedge();
+
+        // public static properties
+        /// <summary>
+        /// Gets an enabled read preference hedge.
+        /// </summary>
+        public static ReadPreferenceHedge Enabled => __enabled;
```

Comment:
Since the default is for hedging to be enabled, perhaps a `Disabled` property be more useful?

Reply:
This isn't a property of the class. 

It's a static property representing a pre-created instance of ReadPreferenceHedge.